css/layout.css

```diff
@@ -45,6 +45,16 @@
 }
 
 @media screen and (min-width: 769px) {
+  .fa-chevron-right {
+    position: relative;
+    left: 2rem;
+  }
+
+  .fa-chevron-left {
+    position: relative;
+    right: 2rem;
+  }
```

Comment:
It's not good to style icons like this because now you can't use these icons for anything else. Instead of adding properties to `.fa-whatever`, make new classes `.next-joke`, `.previous-joke` and put them in your code with the `.fa-whatever` classes.